spec/helpers/application_helper_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe 'ApplicationHelper' do
-  describe "full_title" do
+  describe 'full_title' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.